charts/lnd/values.yaml

```
@@ -86,7 +86,7 @@ lndGeneralConfig:
   - maxchansize=500000000
   - bitcoin.timelockdelta=60
   - default-remote-max-htlcs=50
-  - debuglevel=warn
+  - debuglevel=info
```

if we really want enable info for db compact then we should try something like this:
```suggestion
  - debuglevel=warn,CHDB=info
```
you can get more subsystems with `lncli debuglevel --show`